db/migrate/20221018175449_add_user_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: 'author_id', index: true
```

Comment:
- [ ] Kindly set up the reference to point to the author, instead of user

Reply:
Hi @DammyShittu 
Highly appreciated 
Many thanks for the valuable comment and advice